api/v1alpha1/lvmcluster_webhook.go

```diff
@@ -197,26 +201,25 @@ func (l *LVMCluster) ValidateDelete() (admission.Warnings, error) {
 	return []string{}, nil
 }
 
-func (l *LVMCluster) verifySingleDefaultDeviceClass() error {
+func (l *LVMCluster) verifyDeviceClass() (admission.Warnings, error) {
 	deviceClasses := l.Spec.Storage.DeviceClasses
-	if len(deviceClasses) == 1 {
-		return nil
-	} else if len(deviceClasses) < 1 {
-		return fmt.Errorf("at least one deviceClass is required")
+	if len(deviceClasses) < 1 {
+		return nil, fmt.Errorf("at least one deviceClass is required")
 	}
 	countDefault := 0
 	for _, deviceClass := range deviceClasses {
 		if deviceClass.Default {
 			countDefault++
 		}
 	}
-	if countDefault < 1 {
-		return fmt.Errorf("one default deviceClass is required. Please specify default=true for the default deviceClass")
-	} else if countDefault > 1 {
-		return fmt.Errorf("only one default deviceClass is allowed. Currently, there are %d default deviceClasses", countDefault)
+	if countDefault > 1 {
+		return nil, fmt.Errorf("only one default deviceClass is allowed. Currently, there are %d default deviceClasses", countDefault)
+	}
+	if countDefault == 0 {
+		return admission.Warnings{"no default deviceClass was specified, it will be mandatory to specify the storage class"}, nil
```

Comment:
I think we should clarify where it will be mandatory to specify the storage class. Something like `.. the storage class in the PVCs`.